modules/nw-ovn-kubernetes-matrix.adoc

```diff
@@ -64,7 +64,9 @@ endif::[]
 
 2. Egress router for OVN-Kubernetes supports only redirect mode.
 
-3. IPv6 is supported only on bare metal, vSphere, {ibm-power-name}, {ibm-z-name}, and Red Hat OpenStack clusters.
+3. IPv6 single-stack and dual-stack networking is supported only on bare metal, {ibm-power-name}, {ibm-z-name}, and Red Hat OpenStack clusters.
 
-4. IPv6 single stack is not supported on {ibm-power-name}, {ibm-z-name}, and Red Hat OpenStack clusters.
+4. IPv6 single-stack networking is not supported on {vmw-full}, {ibm-power-name}, {ibm-z-name}, and Red Hat OpenStack clusters.
```

Comment:
What about?

IPv6 single-stack on vSphere was supported for disconnected installation